packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Choice/Choice.spec.js

```diff
@@ -1,125 +1,157 @@
-import { shallow } from 'enzyme'
-import { Button, Carousel } from 'grommet'
-import { types } from 'mobx-state-tree'
+import { expect } from 'chai'
 import React from 'react'
 import sinon from 'sinon'
-import { PrimaryButton } from '@zooniverse/react-components'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
 
 import { task as mockTask } from '@plugins/tasks/SurveyTask/mock-data'
-import { default as Task } from '@plugins/tasks/SurveyTask'
 import Choice from './Choice'
-import ConfusedWith from './components/ConfusedWith'
-import Questions from './components/Questions'
+import en from './locales/en'
 
 describe('Component > Choice', function () {
-  let wrapper
-  let task = Task.TaskModel.create({
-    choices: mockTask.choices,
-    images: mockTask.images,
-    questions: mockTask.questions,
-    questionsMap: mockTask.questionsMap,
-    questionsOrder: mockTask.questionsOrder,
-    taskKey: 'T0',
-    type: 'survey'
-  })
-
-  const handleDeleteSpy = sinon.spy()
-  const onIdentifySpy = sinon.spy()
-
-  before(function () {
-    types.model('MockStore', {
-      task: Task.TaskModel
-    })
-      .create({
-        task
-      })
-
-    wrapper = shallow(
+  it('should render without crashing', function () {
+    render(
       <Choice
         choiceId='KD'
-        handleDelete={handleDeleteSpy}
-        onIdentify={onIdentifySpy}
-        task={task}
+        task={mockTask}
       />
     )
-  })
-
-  it('should render without crashing', function () {
-    expect(wrapper).to.be.ok()
+    expect(screen).to.be.ok()
   })
 
   describe('with choice with images, confusions, and questions', function () {
     // choice 'KD' (Kudu) includes images, confusions, and questions
 
     it('should render Carousel', function () {
-      expect(wrapper.find(Carousel)).to.have.lengthOf(1)
+      render(
+        <Choice
+          choiceId='KD'
+          task={mockTask}
+        />
+      )
+      expect(screen.getByTestId('choice-images')).to.exist()
```

Comment:
Yes, this confused me! Here's what `screen.debug()` (using async/await and waitFor) includes:
<img width="458" alt="Screen Shot 2021-08-19 at 1 51 20 PM" src="https://user-images.githubusercontent.com/12118751/130129275-e9748fcf-9e79-41fa-862d-5269d36e5865.png">
note the `<noscript />` I think is from [the ThumbnailImage component](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-react-components/src/Media/components/ThumbnailImage/ThumbnailImage.js#L93-L97) because the images are taking longer to load than the 1000ms of RTL's waitFor default timeout.